go/vt/topo/etcd2topo/watch.go

```diff
@@ -62,15 +63,35 @@ func (s *Server) Watch(ctx context.Context, filePath string) (*topo.WatchData, <
 	go func() {
 		defer close(notifications)
 
+		var count int
```

Comment:
'count' is a bit confusing as a variable name, not sure what it counts. How about: watchRetries

Reply:
done